package.json

```diff
@@ -1,14 +1,32 @@
 {
   "name": "filenamelint",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "description": "Linter for file and directory names",
   "engines": {
     "node": ">=12.10.0",
     "npm": ">=6.10.3"
   },
-  "main": "index.js",
+  "main": "lib/index.js",
+  "files": [
+    "lib/**/*"
+  ],
   "scripts": {
-    "start": "node index.js"
+    "start": "npm run build:live",
+    "build": "tsc -p .",
+    "build:live": "nodemon --watch 'src/**/*.ts' --exec 'ts-node' src/index.ts",
```

Comment:
why not `tsc --watch --incremental`?

Reply:
nodemon has `--exec` option which is useful now. I am going to remove it later.